docs/planning/backend/app/core/config.py

```
@@ -0,0 +1,42 @@
+
+    # External APIs
+    FRAGELLA_API_KEY: Optional[str] = None
+    FRAGELLA_BASE_URL: str = "https://api.fragella.com/api/v1"
+    FRAGELLA_MONTHLY_LIMIT: int = 20
+
+    # OpenRouter for LLM-powered recommendations
+    OPENROUTER_API_KEY: Optional[str] = None
+    OPENROUTER_DEFAULT_MODEL: str = "anthropic/claude-3.5-sonnet"
+
+    # Security
+    SECRET_KEY: str = "change-me-in-production"
```

Ruff (0.14.10)

Possible hardcoded password assigned to: "SECRET_KEY"

(S105)